src/index.html

```diff
@@ -11,8 +11,45 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="product-cards-container">
+      <div
+        class="product-card"
+        data-qa="card"
+      >
+        <img
+          src="images/imac.jpeg"
+          alt="imac"
+          class="product-card__img"
+        />
+        <p class="product-card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="product-card__code">Product code: 195434</p>
+        <div class="stars stars--4">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star stars__star--inactive"></span>
+          <p class="stars__reviews">Reviews: 5</p>
+        </div>
```

Comment:
1. `reviews` isn't a `stars` elements. it's product one
2. you should use `stars` modifier to show active/inactive stars
```suggestion
        <div class="product-card__rating">
          <div class="stars stars--4">
            <span class="stars__star"></span>
            <span class="stars__star"></span>
            <span class="stars__star"></span>
            <span class="stars__star"></span>
            <span class="stars__star"></span>
          </div>
          
          <p class="product-card__reviews">Reviews: 5</p>
        </div>
```